package.json

```diff
@@ -23,6 +23,9 @@
   },
   "homepage": "https://github.com/fastify/fastify-sensible#readme",
   "devDependencies": {
+    "@types/http-errors": "^1.6.1",
+    "@types/proxy-addr": "^2.0.0",
+    "@types/vary": "^1.1.0",
```

Comment:
It seems these are used in the types. These should be dependencies, but I’m -1 for that. Would you mind just integrating those types with ours?

Reply:
Hi @mcollina , I don't mind doing that. Another alternative is to upload `fastify-sensible` types definition on @types domain (https://github.com/DefinitelyTyped/DefinitelyTyped). What do you think about this option?

Reply:
I'm -1 to that, types on `@types` tend to be of lower quality, harder to maintain, and essentially more problematic than the ones we maintain directly.

Reply:
@mcollina , there is a caveat to that... `@types/proxy-addr` and `@types/vary` depend on `@types/node`, we use in `import * as http from 'http'`. The `@types/node` has a lot of definitions that are used by `http.d.ts`, thus the need to import everyone. Is this a problem? If so, we can either leave `@types/node` as dependency or simply `@types/proxy-addr` and `@types/vary`.

Reply:
`@types/node` is fine, all ts users will add that.

Reply:
@mcollina why not just `peerDependencies` the other packages?

Reply:
npm will install them anyway.